addons/mail/static/src/core/common/message_model.js

```diff
@@ -37,6 +35,8 @@ export class Message extends Record {
     author = Record.one("Persona");
     body = Record.attr("", { html: true });
     composer = Record.one("Composer", { inverse: "message", onDelete: (r) => r.delete() });
+    /** @type {DateTime} */
```

Comment:
Isn't everything Date in js?

Reply:
It the Luxon `DateTime` Object, not the javascript `Date`